### DISCUSSION TITLE
Suggestion: put all language translation folders in a parent "lang" folder

### DISCUSSION THREAD

**Discussion by elisimpson** [BartoszCichecki/LenovoLegionToolkit#1331] [Ideas]
Just to clean up and organize the program directory a bit

**Comment by BartoszCichecki** [comment]
But why? Resources are bundled with respective projects. What's the point of this?

**Reply by elisimpson** [reply]
Just to avoid having dozens of language folders in the main program dir rather than just one folder. It's not important, just a suggestion as I typically see most software being packaged that way.

**Reply by BartoszCichecki** [reply]
Well, this is the standard way msbuild outputs binaries and resources. I don't see a reason to mess with it.
